README.md

```diff
@@ -44,17 +44,19 @@ Please refer to the [Contribution Guidelines](./.github/CONTRIBUTING.md) before
 
 We recommend the use of [React Storybook](https://github.com/storybooks/react-storybook) for developing components.
 
-1. Start the server:
+1. (Optional) Set `true` to `CARBON_USE_EXPERIMENTAL_FEATURES` environment variable to test some of the experimental features.
+
```

Comment:
Can we add the actual code sample here they would need to use, in case someone isn't familiar with environment variables. 

`CARBON_USE_EXPERIMENTAL_FEATURES=true yarn storybook`

Reply:
Done - Thanks @alisonjoseph for your feedback!